backends/lean/Base/Progress/Progress.lean

```diff
@@ -219,7 +209,7 @@ def getFirstArg (args : Array Expr) : Option Expr := do
    Return true if it succeeded. -/
 def tryLookupApply (keep : Option Name) (ids : Array (Option Name)) (splitPost : Bool)
```

Comment:
I think we should rename this function to `tryApply`, because it doesn't perform a lookup anymore.
Also, it might make sense to move the match over the option (and the comments printed in the trace are not valid anymore I believe).